src/main/java/com/cgdpd/library/dto/pagination/SortParam.java

```diff
@@ -0,0 +1,25 @@
+package com.cgdpd.library.dto.pagination;
+
+import static com.cgdpd.library.validation.Validator.required;
+
+import org.springframework.data.domain.Sort;
+
+public record SortParam(String field,
+                        Direction direction) {
+
+    public SortParam(String field, Direction direction) {
+        this.field = required("field", field);
+        this.direction = required("order", direction);
+    }
+
+    public enum Direction {
```

Comment:
This should be in the bottom, make sure to execute reformat code shortcut often
`⌥⇧⌘L (macOS), or Ctrl+Alt+Shift+L (Windows/Linux)`